src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEMethodSymbol.cs

```diff
@@ -233,6 +233,7 @@ private sealed class UncommonFields
             public ImmutableArray<string> _lazyNotNullMembers;
             public ImmutableArray<string> _lazyNotNullMembersWhenTrue;
             public ImmutableArray<string> _lazyNotNullMembersWhenFalse;
+            public MethodSymbol _lazyExplicitClassOverride;
```

Comment:
If this is allowed to contain null consider enabling nullable for this piece so that you can annotate it.

Reply:
All of these fields that are of ref type can be null. The thing to annotate would be `internal MethodSymbol ExplicitlyOverriddenClassMethod`. But I don't want to take on annotating this in the context of this PR.

---
In reply to: [437685349](https://github.com/dotnet/roslyn/pull/44025#discussion_r437685349) [](ancestors = 437685349)